admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php

```diff
@@ -506,29 +501,15 @@ public function submit_request() {
 			}
 			$message .= '</body>
                 </html>';
-			add_filter( 'wp_mail_content_type', create_function( '', 'return "text/html";' ) );
+			add_filter(
+				'wp_mail_content_type',
+				function() {
+					return 'text/html';
+				}
+			);
 			$headers       = 'From: ' . $form_data['name'] . ' <' . $form_data['email'] . '>' . "\r\n";
-			if ( wp_mail( RT_HELPDESK_SUPPORT_EMAIL, '[Helpdesk] ' . $mail_type . ' from ' . str_replace( array(
-					'http://',
-					'https://'
-				), '', $form_data['website'] ), stripslashes( $message ), $headers ) ) {
-
-//				echo '<div class="rtbiz_hd-success" style="margin:10px 0;">';
-//				if ( 'new_feature' == $form_data['request_type'] ) {
-//					echo '<p>' . __( 'Thank you for your Feedback/Suggestion.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//				} else {
-//					echo '<p>' . __( 'Thank you for posting your support request.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//					echo '<p>' . __( 'We will get back to you shortly.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//				}
-//				echo '</div>';
-//			} else {
-//				echo '<div class="rtbiz_hd-error">';
-//				echo '<p>' . __( 'Your server failed to send an email.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//				echo '<p>' . __( 'Kindly contact your server support to fix this.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//				echo '<p>' . sprintf( __( 'You can alternatively create a support request <a href="%s">here</a>', RTBIZ_HD_TEXT_DOMAIN ), 'https://rtcamp.com/premium-support/' ) . '</p>';
-//				echo '</div>';
-			}
-//			die();
+			$support_email = 'support@rtcamp.com';
```

Comment:
There seems to be a hardcoded email Id for support email?

Reply:
@milindmore22 

It is same from the start. 

https://github.com/rtCamp/rtbiz-helpdesk/commit/7ea9b7ae65ff8c4a29e23950c868778964f45a53#diff-2b419e1f76958d640efa051d8e7d92e1a2453ab985e595ab40b4f4d2a2370ea2